### PR TITLE
Inventory History Detail screen

### PR DESCRIPTION
**Pull request by nvs266**

### CODE REVIEW

File: app/src/main/java/com/framgia/fdms/data/source/InventoryHistoryDataSoure.java [hunk 1]
```diff
@@ -11,4 +13,6 @@
 
 public interface InventoryHistoryDataSoure {
     Observable<List<String>> getInventoryHistoryByUserId(int userId);
+
+    Observable<List<Device>> getCheckedHistoryDetail(int currentUserId, int checkedUserId);
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fdms/data/source/InventoryHistoryDataSoure.java]: currentUserId có thể  suy ra được từ token nên k cần cái này

Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fdms/data/source/InventoryHistoryDataSoure.java]: checkedUserId > staffId

File: app/src/main/java/com/framgia/fdms/screen/inventoryhistorydetail/InventoryHistoryDetailActivity.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import com.framgia.fdms.data.source.remote.InventoryHistoryRemoteDataSource;
+import com.framgia.fdms.databinding.ActivityInventoryHistoryDetailBinding;
+
+public class InventoryHistoryDetailActivity extends AppCompatActivity {
+
+    private static final String EXTRA_CHECKED_USER = "EXTRA_CHECKED_USER";
+
+    private InventoryHistoryDetailViewModel mViewModel;
+
+    public static Intent getCheckedUserIdIntent(Context context, User checkedUser) {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fdms/screen/inventoryhistorydetail/InventoryHistoryDetailActivity.java]: checkedUser > user

File: app/src/main/java/com/framgia/fdms/screen/inventoryhistorydetail/InventoryHistoryDetailAdapter.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    public InventoryHistoryDetailAdapter(@NonNull Context context,
+                                         InventoryHistoryDetailViewModel viewModel,
+                                         List<Device> devices) {
+        super(context);
+        mViewModel = viewModel;
+        mDevices = devices;
+    }
+
+    @Override
+    public void addItem(List<Device> devices) {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fdms/screen/inventoryhistorydetail/InventoryHistoryDetailAdapter.java]: check devices ==null reutnr

File: app/src/main/java/com/framgia/fdms/screen/inventoryhistorydetail/InventoryHistoryDetailPresenter.java [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+ * Created by sonng266 on 11/04/2018.
+ */
+
+public class InventoryHistoryDetailPresenter implements InventoryHistoryDetailContract.Presenter {
+
+    private InventoryHistoryDetailViewModel mViewModel;
+    private UserRepository mUserRepository;
+    private InventoryHistoryRepository mInventoryHistoryRepository;
+    private CompositeDisposable mSubscription;
+    private User mCheckedUser;
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fdms/screen/inventoryhistorydetail/InventoryHistoryDetailPresenter.java]: rename to user

File: app/src/main/java/com/framgia/fdms/data/source/InventoryHistoryDataSoure.java (hunk 1 above)
Comment (nvs266) [reply app/src/main/java/com/framgia/fdms/data/source/InventoryHistoryDataSoure.java]: (Resolved)

Comment (nvs266) [reply app/src/main/java/com/framgia/fdms/data/source/InventoryHistoryDataSoure.java]: (Resolved)

File: app/src/main/java/com/framgia/fdms/screen/inventoryhistorydetail/InventoryHistoryDetailActivity.java (hunk 2 above)
Comment (nvs266) [reply app/src/main/java/com/framgia/fdms/screen/inventoryhistorydetail/InventoryHistoryDetailActivity.java]: (Resolved)

File: app/src/main/java/com/framgia/fdms/screen/inventoryhistorydetail/InventoryHistoryDetailAdapter.java (hunk 3 above)
Comment (nvs266) [reply app/src/main/java/com/framgia/fdms/screen/inventoryhistorydetail/InventoryHistoryDetailAdapter.java]: (Resolved)

File: app/src/main/java/com/framgia/fdms/screen/inventoryhistorydetail/InventoryHistoryDetailPresenter.java (hunk 4 above)
Comment (nvs266) [reply app/src/main/java/com/framgia/fdms/screen/inventoryhistorydetail/InventoryHistoryDetailPresenter.java]: (Resolved)

**Pull request merged**